charts/newrelic-infrastructure/templates/controlplane/_tolerations_helper.tpl

```diff
@@ -0,0 +1,25 @@
+{{- /*
+Defaults for controlPlane while keeping then overridable.
+*/ -}}
+{{- define "nriKubernetes.controlPlane.tolerations.defaults" -}}
+- operator: "Exists"
+  effect: "NoSchedule"
+- operator: "Exists"
+  effect: "NoExecute"
+{{- end -}}
+
+
+
+{{- /*
+As this chart deploys what it should be three charts to maintain the transition to v3 as smooth as possible.
+This means that this chart has 3 tolerations so a helper should be done per scraper.
+*/ -}}
+{{- define "nriKubernetes.controlPlane.tolerations" -}}
```

Comment:
oh this is tricky. So precedence is like CP-toletartion -> chart toleration -> global toleration -> default toleration. 
NIT: Perhaps adding something like this to the comment could help to understand it faster

Reply:
cool there are test for this !! 👏

Reply:
I leave this to the nit-picking commit.

Reply:
The latest version is so much simpler than the one you reviewed here.

I think no further explanation is needed. WDYT?